src/types/stationPerfecture.ts

```diff
@@ -0,0 +1,5 @@
+export type StationPerfecture = {
+    id: number;
+    label: string;
+    value?: string;
```

Comment:
do we use the `value` in our code?

Reply:
At the moment, no. Do you think it should be removed?

Reply:
yeahh it's just a thought tbf - if we don't really use/need it in the code then should we remove it and keep the code simpler and cleaner? I'm not sure yet what's the best practice to be honest, should we keep the data strcture as returned from the API and describe the whole object, or instead keep only what we need in the code?

Reply:
I am going to remove it, but I will keep the id which is also not used (I do have a case for filtering where the id might become handy).